src/tests/receiving/receiveToHoldBin.test.ts

```diff
@@ -0,0 +1,177 @@
+import { ShipmentType } from '@/constants/ShipmentType';
+import { expect, test } from '@/fixtures/fixtures';
+import { StockMovementResponse } from '@/types';
+import UniqueIdentifier from '@/utils/UniqueIdentifier';
+
+test.describe('Receive item into hold bin', () => {
+  let STOCK_MOVEMENT: StockMovementResponse;
+  const uniqueIdentifier = new UniqueIdentifier();
+  const binLocationName = uniqueIdentifier.generateUniqueString('holdbin');
+
+  test.beforeEach(
+    async ({
+      supplierLocationService,
+      mainLocationService,
+      stockMovementService,
+      mainProductService,
+      page,
+      locationListPage,
+      createLocationPage,
+    }) => {
+      const supplierLocation = await supplierLocationService.getLocation();
+      const mainLocation = await mainLocationService.getLocation();
+      const PRODUCT_ONE = await mainProductService.getProduct();
+
+      STOCK_MOVEMENT = await stockMovementService.createInbound({
+        originId: supplierLocation.id,
+      });
+
+      await stockMovementService.addItemsToInboundStockMovement(
+        STOCK_MOVEMENT.id,
+        [
+          { productId: PRODUCT_ONE.id, quantity: 20 },
+        ]
+      );
+
+      await stockMovementService.sendInboundStockMovement(STOCK_MOVEMENT.id, {
+        shipmentType: ShipmentType.AIR,
+      });
+
+      await test.step('Create bin location with hold stock activity for location', async () => {
+        await page.goto('./location/list');
+        await locationListPage.searchByLocationNameField.fill(
+          mainLocation.name
+        );
+        await locationListPage.findButton.click();
+        await expect(
+          locationListPage.getLocationEditButton(mainLocation.name)
+        ).toBeVisible();
+        await locationListPage.getLocationEditButton(mainLocation.name).click();
+        await createLocationPage.binLocationTab.click();
+        await page.waitForTimeout(1000);
```

Comment:
As I see more and more of these timeouts, I dislike it more. Let's reiterate tomorrow what is the reason for these, and see what can be improved to avoid these.